scripts/github-actions/update-browser-versions.js

```diff
@@ -100,7 +100,7 @@ const updatePRTitle = async ({ context, github, baseBranch, branchName, descript
     return
   }
 
-  await github.pulls.update({
```

Comment:
Should we update https://github.com/cypress-io/cypress/blob/develop/.github/workflows/semantic-pull-request.yml#L24 as well to get everything in sync? might be out of scope really

Reply:
I went ahead and updated this

Reply:
Your the best!

Reply:
It broke the semantic checker on this PR, but I think that's because it's not actually using the action from this PR but still using the code. I'll probably merge this but I'll be prepared to back it out and rethink if this is problematic.